README.md

```diff
@@ -1,10 +1,35 @@
-# EU eHealthNetwork Digitial COVID Certificate value sets
+# EU eHealthNetwork Digital COVID Certificate value-sets
```

Comment:
"value-sets" or "value sets" (or "valuesets")? Any of these are fine, as long as it's consistent across the entire repo.

Reply:
I changed it al to value-sets, but am happy with valuesets.

Reply:
Then keep it at "value-sets"!